server/chat-plugins/helptickets.ts

```diff
@@ -868,27 +853,16 @@ export const pages: PageTable = {
 			buf += `<table style="margin-left: auto; margin-right: auto"><tbody><tr><th colspan="5"><h2 style="margin: 5px auto">${this.tr`Help tickets`}</h1></th></tr>`;
 			buf += `<tr><th>${this.tr`Status`}</th><th>${this.tr`Creator`}</th><th>${this.tr`Ticket Type`}</th><th>${this.tr`Claimed by`}</th><th>${this.tr`Action`}</th></tr>`;
 
-			const keys = Object.keys(tickets).sort((aKey, bKey) => {
-				const a = tickets[aKey];
-				const b = tickets[bKey];
-				if (a.open !== b.open) {
-					return (a.open ? -1 : 1);
-				}
-				if (a.open) {
-					if (a.active !== b.active) {
-						return (a.active ? -1 : 1);
-					}
-					return a.created - b.created;
-				}
-				return b.created - a.created;
-			});
+			const sortedTickets = Utils.sortBy(Object.values(tickets), ticket => [
+				ticket.open,
+				ticket.open ? [ticket.active, ticket.created] : -ticket.created,
+			]);
```

Comment:
This is a good representative sample of the kind of code that `sortBy` massively simplifies and makes more readable.

I hope it's as easy for everyone else as it is for me to read this as "open active tickets by earliest first, then open inactive tickets by earliest first, then closed tickets by earliest last".